contrib/sarplus/python/pysarplus/SARModel.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
```

Comment:
@miguelgfierro Is it the correct license header for Python code?

Reply:
yes